src/components/QuickOpenModal.js

```diff
@@ -398,9 +388,11 @@ export class QuickOpenModal extends Component<Props, State> {
           selectedItemId={
             expanded && items[selectedIndex] ? items[selectedIndex].id : ""
           }
+          isLoading={
+            (this.isFunctionQuery() || this.isVariableQuery()) && symbolsLoading
+          }
```

Comment:
do you think we could re-purpose the summaryMsg?

If not, could we add a new generic field like  `secondaryLabel`? It would be nice to keep this input generic.